src/optimize/base_optimizer.js

```diff
@@ -101,40 +135,24 @@ class BaseOptimizer {
 
       module: {
         loaders: [
-          {
-            test: /\.less$/,
-            loader: ExtractTextPlugin.extract(
-              'style',
-              `css${mapQ}!autoprefixer${mapQPre}{ "browsers": ["last 2 versions","> 5%"] }!less${mapQPre}dumpLineNumbers=comments`
-            )
-          },
-          {
-            test: /\.scss$/,
-            loader: ExtractTextPlugin.extract(
-              'style',
-              `css${mapQ}!autoprefixer${mapQPre}{ "browsers": ["last 2 versions","> 5%"] }!sass${mapQPre}`
-            )
-          },
-          { test: /\.css$/, loader: ExtractTextPlugin.extract('style', `css${mapQ}`) },
+          { test: /\.less$/, loader: makeStyleLoader('less') },
+          { test: /\.scss$/, loader: makeStyleLoader('sass') },
+          { test: /\.css$/, loader: makeStyleLoader() },
           { test: /\.jade$/, loader: 'jade' },
           { test: /\.json$/, loader: 'json' },
           { test: /\.(html|tmpl)$/, loader: 'raw' },
-          { test: /\.png$/, loader: 'url?limit=10000&name=[path][name].[ext]' },
-          { test: /\.(woff|woff2|ttf|eot|svg|ico)(\?|$)/, loader: 'file?name=[path][name].[ext]' },
-          { test: /[\/\\]src[\/\\](core_plugins|ui)[\/\\].+\.js$/, loader: `rjs-repack${mapQ}` },
+          { test: /\.png$/, loader: 'url' },
```

Comment:
The change in #6177 adds a resolver alias function, which triggers [this bug](https://github.com/webpack/webpack/issues/1289). I assumed that the limit and name query  parameters weren't being used, but if the build output suggests otherwise then we should put them back